tests/wpt/meta-legacy-layout/css/css-paint-api/background-image-alpha.https.html.ini

```diff
@@ -1,2 +1,2 @@
 [background-image-alpha.https.html]
-  expected: CRASH
+  expected: PASS
```

Comment:
Some of these updated expectations are being marked as PASS rather than being removed. Is that intentional?